src/mappings/rewardsManager.ts

```diff
@@ -18,7 +18,13 @@ export function handleRewardsAssigned(event: RewardsAssigned): void {
   let indexerIndexingRewards = event.params.amount
     .times(BigInt.fromI32(indexer.indexingRewardCut))
     .div(BigInt.fromI32(1000000))
-  let delegatorIndexingRewards = event.params.amount.minus(indexerIndexingRewards)
+
+  // If the delegation pool has zero tokens, the contracts don't give away any rewards
+  let delegatorIndexingRewards =
```

Comment:
Should the `indexerIndexingRewards` also be calculated as if it was receiving 100% of the rewards?

Reply:
good catch, need to fix it